easybuild/easyconfigs/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libutempter'
+version = '1.1.6.2'
+
+homepage = 'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git'
+description = """libutempter is library that provides an interface for terminal
+ emulators such as screen and xterm to record user sessions to utmp and wtmp files."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+source_urls = [
+    'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h=' +
+    version[0:5] + '-alt' + version[6:7] + ';sf=tgz']
```

Comment:
@SimonPinches If this doesn't work and there's no alternative for downloading the sources, we'll need to escape with a manual download for now (but then that should be clarified with a comment that mentions the full URL).

Reply:
@boegel I've added the full download link in a comment.  Perhaps you can help make the auto-download work but so far I've failed :-(